connection.go

```diff
@@ -17,7 +17,7 @@ import (
 	"strconv"
 	"time"
 
-	"github.com/vesoft-inc/fbthrift/thrift/lib/go/thrift"
+	"github.com/apache/thrift/lib/go/thrift"
```

Comment:
Sorry that we cannot replace the thrift with apache thrift, we need to keep the rpc protocol the same with the server to avoid some incompatibility issues.